sentry-android-core/src/main/java/io/sentry/android/core/SpanFrameMetricsCollector.java

```diff
@@ -1,43 +1,82 @@
 package io.sentry.android.core;
 
+import androidx.annotation.NonNull;
```

Comment:
let's use `org.jetbrains.annotations.NotNull;` as everywhere else

```suggestion
import org.jetbrains.annotations.NotNull;
```